core/internal/cltest/cltest.go

```diff
@@ -1648,3 +1648,17 @@ func MustGetStateForKey(t testing.TB, kst keystore.Eth, key ethkey.KeyV2) ethkey
 func NewBulletproofTxManagerORM(t *testing.T, db *sqlx.DB, cfg pg.LogConfig) bulletprooftxmanager.ORM {
 	return bulletprooftxmanager.NewORM(db, logger.TestLogger(t), cfg)
 }
+
+// ClearDBTables deletes all rows from the given tables
+func ClearDBTables(t *testing.T, db *sqlx.DB, tables ...string) {
+	tx, err := db.Beginx()
+	require.NoError(t, err)
+
+	for _, table := range tables {
+		_, err = tx.Exec(fmt.Sprintf("DELETE FROM %s", table))
+		require.NoError(t, err)
+	}
+
+	err = tx.Commit()
+	require.NoError(t, err)
```

Comment:
This is nice and simple, but you might consider using `pg.SqlxTransaction` for timeouts, panic handling etc.

Reply:
I decided to keep it simple because this is for tests and usually tests have their specific timeouts.